awx/main/tasks.py

```diff
@@ -1232,6 +1238,9 @@ def run(self, pk, **kwargs):
                 status = 'failed'
                 extra_update_fields['job_explanation'] = self.instance.job_explanation
 
+        except InvalidVenvError as e:
+            tb = e.message
```

Comment:
I don't think we should put this message in _this_ field.

You could, maybe, do

```python
extra_update_fields['job_explanation'] = e.message
```

Then, status will be "error", and the UI will have a `job_explanation` message to show. I find this to be very similar to how we handle similar states when they happen in the task manager.